Where_Is_My_Wife/Assets/_Main/Scripts/UI/Settings/LanguageConfig.cs

```diff
@@ -0,0 +1,22 @@
+using WhereIsMyWife.Managers;
+
+namespace WhereIsMyWife.Config
+{
+    public class LanguageConfig : ConfigSelection
```

Comment:
Not needed

Reply:
96665c08c08aa3776ec96d7030a7f19c1066d17c